### PR TITLE
feat(docs):[TRI-1585] Add missing components to building block view

### PR DESCRIPTION
**Pull request by ds-jhartmann**

### CODE REVIEW

File: docs/src/docs/arc42/building-block-view/whitebox-overall.adoc [hunk 1]
```diff
@@ -31,4 +31,16 @@ include::../../../uml-diagrams/building-block-view/whitebox_overall_decentral.pu
...
+|The Discovery Finder is used to get the EDC Discovery Endpoint for a certain type of identification, e.g. BPNL.
+
+|EDC Discovery
+|The EDC Discovery Service is used to get EDC endpoints for a certain BPNL.
+
+|Semantic Hub
+|Semantic Hub provides information about semantic models in a specific version. This also includes JSON schemas, which IRS uses to validate the payload received by data providers.
+
+|BPDM
+|The Business Partner Data Management (BPDM) Service is used by the IRS to access shared business partner data for BPNLs of DAta Providers along the value chain.
```
Comment (mkanal) [inline docs/src/docs/arc42/building-block-view/whitebox-overall.adoc:45]:
```suggestion
|The Business Partner Data Management (BPDM) Service is used by the IRS to access shared business partner data for BPNs of Data Providers along the supply chain.
```

File: docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml [hunk 2]
```diff
@@ -2,51 +2,66 @@
...
+IrsApiConsumer -left(0- IRS : **[01]**
+IrsApiConsumer <.r.> IAM_IRS  : **[02]**
+IAM_IRS  <..> Portal_IdP  : **[03]**
+EDC <-> SubmodelServerOEM  : **[04]**
+EDC <->  DT_REG_OEM  : **[05]**
+IRS <.d.> EDC  : **[06]**
+IRS <..> IAM_IRS : **[07]**
+IRS <..> edc_discovery  : **[08]**
+
+IRS <..> discovery_finder  : : **[09]**
```
Comment (ds-jhartmann) [inline docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml:52]:
```suggestion
IRS <..> discovery_finder  : **[09]**
```

File: docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml [hunk 3]
```diff
@@ -2,51 +2,66 @@
...
+IRS <..> semantic_hub  : : **[10]**
```
Comment (ds-jhartmann) [inline docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml:53]:
```suggestion
IRS <..> semantic_hub  : **[10]**
```

File: docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml [hunk 4]
```diff
@@ -2,51 +2,66 @@
...
+IRS <..> bpdm  : : **[11]**
```
Comment (ds-jhartmann) [inline docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml:54]:
```suggestion
IRS <..> bpdm  : **[11]**
```

File: docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml [hunk 5]
```diff
@@ -2,51 +2,66 @@
...
+EDC <..> miw  : : **[12]**
```
Comment (ds-jhartmann) [inline docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml:55]:
```suggestion
EDC <..> miw  : **[12]**
```

File: docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml [hunk 6]
```diff
@@ -2,51 +2,66 @@
...
+EDC <..> EDCProvider  : : **[13]**
```
Comment (ds-jhartmann) [inline docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml:56]:
```suggestion
EDC <..> EDCProvider  : **[13]**
```

File: docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml [hunk 7]
```diff
@@ -2,51 +2,66 @@
...
+
+miw <..> EDCProvider  : : **[14]**
```
Comment (ds-jhartmann) [inline docs/src/uml-diagrams/building-block-view/whitebox_overall_decentral.puml:58]:
```suggestion
miw <..> EDCProvider  : **[14]**
```

File: docs/src/docs/arc42/building-block-view/whitebox-overall.adoc [hunk 8]
```diff
@@ -31,4 +31,72 @@ include::../../../uml-diagrams/building-block-view/whitebox_overall_decentral.pu
...
+| IRS MUST authorize at central IAM
+
+| 08
+| IRS lookup for EDC Provider by given BPNs over the **EDC Discovery Service**
+
+| 09
+| IRS lookup for **EDC Discovery Services** by given type <BPN> over the **Discovery Finder**
+
+| 10
+| IRS uses **Semantic Hub** to validate  of **SubmodelAspects** payloads agains the schema provided in **Semantic Hub**
```
Comment (mkanal) [inline docs/src/docs/arc42/building-block-view/whitebox-overall.adoc:80]:
```suggestion
| IRS uses **Semantic Hub** to validate of **SubmodelAspects** payloads agains the schema provided in **Semantic Hub**
```

File: docs/src/docs/arc42/building-block-view/whitebox-overall.adoc [hunk 9]
```diff
@@ -31,4 +31,72 @@ include::../../../uml-diagrams/building-block-view/whitebox_overall_decentral.pu
...
+
+| 11
+| In case "lookupBPNs" is active IRS provides a lookup of company for given BPN
+
+| 12
+| **EDC** is connected to **Managed Identity Wallet** for access policy check for data offers
+
+| 13
+| **EDC** communication covering negotation and data consumption
```
Comment (mkanal) [inline docs/src/docs/arc42/building-block-view/whitebox-overall.adoc:89]:
```suggestion
| **EDC** communication covering negotiation and data consumption
```

**Review by mkanal — APPROVED** [review]

File: docs/src/docs/arc42/runtime-view/overall.adoc [hunk 10]
```diff
@@ -1,4 +1,8 @@
 = Overall
+skinparam monochrome true
+skinparam shadowing false
+skinparam nodesep 100
+skinparam ranksep 20
```
Comment (ds-jhartmann) [inline docs/src/docs/arc42/runtime-view/overall.adoc:5]: This does not belong in a adoc file
```suggestion
```

File: docs/src/docs/arc42/building-block-view/whitebox-overall.adoc [hunk 11]
```diff
@@ -31,4 +31,72 @@ include::../../../uml-diagrams/building-block-view/whitebox_overall_decentral.pu
...
+
+| 14
+| **EDC** is connected to **Managed Identity Wallet** for access policy check for data offers
+
+| 15
+| **IRS** accessing to **SubmodelServer** on Tier Level using the **EDC**
+
+| 16
+| **IRS** accessing the **decentral DigitalTwinRegistry** on Tier Level using the **EDC**
+
```
Comment (ds-jhartmann) [inline docs/src/docs/arc42/building-block-view/whitebox-overall.adoc:99]: Unterminated Table block
```suggestion
|===
```

**Review by mkanal — APPROVED** [review]

**Pull request merged**